chatterbot/corpus/data/english/humor.corpus.json

```diff
@@ -0,0 +1,63 @@
+{
+    "humor": [
+        [
+            "JOKE",
+            "Did you hear the one about the Mountain Goats in the Andes? It was Ba a a a a a d.",
+            "I never forget a face, but in your case I'll make an exception.",
```

Comment:
I don't believe the format of this file would train an instance of ChatterBot properly. Each of these strings is a single statement, but none of them are related to each other. ChatterBot's corpus trainer expects the list to represent a conversation.